users/tests/conftest.py

```diff
@@ -78,11 +78,14 @@ def make_instance(**args):
         args.setdefault('name', get_random_string())
         args.setdefault('client_type', 'public')
         args.setdefault('client_id', get_random_string())
-        args.setdefault('response_type', 'id_token token')
         args.setdefault('redirect_uris', None)
 
+        response_types = args.pop('response_types', ['id_token token'])
```

Comment:
Should it be `['id_token', 'token']` instead (a list with two items rather than one)?

Reply:
@suutari-ai no, oidc-provider began supporting multiple response types per client in 0.7, and each response type is identified by a string which comes straight from the oidc standard.

We are not using the multiple response types feature in our code or tests, it's just that the ORM level API has changed, and this fixture now supports multiple response types if we need them.

Reply:
Ah, then this is OK.